Sources/Server/Request.swift

```diff
@@ -0,0 +1,26 @@
+public struct Request {
+
+     let method: String
+     let path: String
+     var queries: [String: String] = ["greeting": "World"]
+
+     init(method: String, path: String) {
+         self.method = method
+         self.path = path
+     }
+
+     init(method: String, path: String, queries: [String: String]) {
+         self.method = method
+         self.path = path
+         self.queries = sortHelloQueries(queries: queries)
+     }
+
+     func sortHelloQueries(queries: [String: String]) -> [String: String] {
```

Comment:
Not sure something so route specific belongs in here.

Reply:
Yeah, this was a tough one for me, and I didn't know what would be the best way to handle this. It didn't seem right to put it in the Parser and previously it was in Response in a different format.  You think I should put it back in Response?

Reply:
That's a good question. I'm not really sure where it belongs. Something worth talking about is making Response 'dumber', acting more as a data structure and a different abstraction to hold handlers. Curious what @cdemyanovich thinks about that idea. We can talk more at standup tomorrow!

Reply:
Agree that this _eventually_ belongs elsewhere. Where? I'll share some ideas on Tuesday. Perhaps we can talk after my morning project meeting(s). For now, I'll drop a hint: this feels like application logic instead of server logic. :-) Think about how you might separate the two.